spec/genre_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './genre'
+
+describe Genre do
+  it 'should have name' do
+    genre1 = Genre.new('Lamhe')
+    expect(genre1.name).to eq('Lamhe')
+  end
+end
```

Comment:
- [ ] As stated in the requirements, all methods inside classes should be tested. So please ensure to add tests for the `add_item` methods from the `Gender` class. 👌

### Requirement :point_down: 
![Screenshot from 2023-11-03 14-59-27](https://github.com/dvcorreia1984/ruby-group-capstone/assets/89228743/62dd0196-4c4f-4871-86a0-c8f997697f3b)